2016-11-25-shiny-1.md

```diff
@@ -348,15 +348,15 @@ tags$div(style="color:red",
   tags$a(href = "http://ourcodingglub.github.io", "Coding Club")
 ```
 
-This creates a block of text that is coloured red (`style="color:red"`), within that block there is a paragraph of text (`tags$p("Visit us at?:")`) and a link (`tags$a(href = "http://ourcodingglub.github.io", "Coding Club")`).
+This creates a block of text that is coloured red (`style="color:red"`), within that block there is a paragraph of text (`tags$p("Visit us at:")`) and a link (`tags$a(href = "https://ourcodingclub.github.io", "Coding Club")`).
 
 For more information on the arguments that can be included in popular Shiny HTML tags, RStudio have a nice wiki at <https://shiny.rstudio.com/articles/tag-glossary.html>.
 
 []()
 
 ### Exporting a finished app
 
-It is easy to send a shiny app to somebody else who also has RStudio. app.R can be sent alongside any data and other resources in a zip file to be unzipped by the recipient and run through R.
+It is easy to send a Shiny app to somebody else who also has RStudio. `app.R` can be sent alongside any data and other resources in a zip file to be unzipped by the recipient and run through R.
```

Comment:
I think adding how to host the app online so that people can view it without R would be good. Then, we can market Shiny as a way to bridge the gap between R users and non-R people, could be useful when e.g. you use R, but your supervisor/co-authors don't - sending them an url with the ready product means they can click away and see the results without ever downloading R. So we could link to shinyapp.io and explain how to use the app hosting service.